src/convolution.c

```diff
@@ -440,6 +460,15 @@ enum qnnp_status qnnp_setup_convolution2d_nhwc_q8(
   const size_t output_height = convolution->output_height;
   const size_t output_width = convolution->output_width;
   if (convolution->flags & QNNP_CONVOLUTION_FLAG_DW) {
+    if (kernel_size == 25) {
+      size_t multipass_acc_size = sizeof(int32_t) * convolution->output_width * convolution->output_height * output_pixel_stride;
```

Comment:
why is the accumulator buffer size proportional to output size?
also, the buffer size should be proportional to `output_channels`, not `output_pixel_stride` (`output_pixel_stride` = `output_channels` + optional padding, e.g. when we process a subrange of channels from a larger tensor)

Reply:
In my kernel, I use 1 int32_t acc per output pixel per channel. And I dont want to share them between multi threads. Let me know how we can improve

Reply:
One alternative is to allocate buffer (of size `output_channels`) on stack, inside `compute_q8dw_multipass`

Reply:
what benefit would stack one bring?

Reply:
on-stack memory is more likely to be in cache, allocation is almost free, and no explicit deallocation needed